pylintrc

```diff
@@ -13,6 +13,7 @@ profile=no
 # Add files or directories to the blacklist. They should be base names, not
 # paths.
 ignore=migrations
+disable=logging-format-interpolation
```

Comment:
why do we need to add this?

Reply:
It seems like this is just a warning, not error, so I think we should remove this line and ignore the warning.